src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -116,11 +116,9 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(GenerateNativeVersionInfo)'=='true'">
```

Comment:
After taking a closer look, it seems that this PropertyGroup is never used so I'm cleaning it up and using it for this PR's purposes